### PR TITLE
qTip2 - Update package.json - npmName + npmFileMap + homepage

### PR DESCRIPTION
**Pull request by Craga89**
Add auto-updating `npmName` and `npmFileMap` entries

### CODE REVIEW

File: ajax/libs/qtip2/package.json [hunk 1]
```diff
@@ -20,5 +20,13 @@
     "type": "git",
     "url": "https://github.com/qTip2/qTip2.git"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "npmName": "qtip2",
+  "npmFileMap": [{
+    "basePath": "/",
+    "files": [
+      "*.js", "*.css", "*.map",
```
Comment (PeterDaveHello) [inline ajax/libs/qtip2/package.json]: You can use `*.+(js|css|map)` here.

File: ajax/libs/qtip2/package.json [hunk 2]
```diff
@@ -20,5 +20,13 @@
...
+      "basic/*.js", "basic/*.css", "basic/*.map"
```
Comment (PeterDaveHello) [inline ajax/libs/qtip2/package.json]: You can use `basic/*.+(js|css|map)` here.

File: ajax/libs/qtip2/package.json [hunk 3]
```diff
@@ -20,5 +20,13 @@
...
+    ]
+  }]
```
Comment (PeterDaveHello) [inline ajax/libs/qtip2/package.json]: Should be :
```js
  }
]
```

File: ajax/libs/qtip2/package.json [hunk 4]
```diff
@@ -20,5 +20,13 @@
     "type": "git",
     "url": "https://github.com/qTip2/qTip2.git"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "npmName": "qtip2",
+  "npmFileMap": [{
```
Comment (PeterDaveHello) [inline ajax/libs/qtip2/package.json]: Should be :
```js
npmFileMap": [
  {
```

File: ajax/libs/qtip2/package.json [hunk 5]
```diff
@@ -3,7 +3,7 @@
   "filename": "jquery.qtip.min.js",
   "version": "2.2.1",
   "description": "Introducing... qTip2. The second generation of the advanced qTip plugin for the ever popular jQuery framework.",
-  "homepage": "http://craigsworks.com/projects/qtip2/",
+  "homepage": "http://qtip2.com",
```
Comment (PeterDaveHello) [inline ajax/libs/qtip2/package.json]: Please also mention this change in commit message.

**Comment by PeterDaveHello** [comment]
@Craga89 would you please mention the library's name in the first line of commit message? Thanks.

**Comment by Craga89** [comment]
Done, thanks @PeterDaveHello

**Comment by PeterDaveHello** [comment]
Please also squash them into only one commit as our document says, thanks!

File: ajax/libs/qtip2/package.json [hunk 6]
```diff
@@ -20,5 +20,15 @@
     "type": "git",
     "url": "https://github.com/qTip2/qTip2.git"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "npmName": "qtip2",
+  "npmFileMap": [
+    {
+      "basePath": "/",
```
Comment (PeterDaveHello) [inline ajax/libs/qtip2/package.json]: The basePath seems to be `dist` in this case.

**Comment by PeterDaveHello** [comment]
I just took a look at its npm package, I'll respectfully suggest you to change to filemap to only one line:
```js
"**/*"
```

**Comment by Craga89** [comment]
Like so?

```js
"npmFileMap": [
	{
		"basePath": "/dist",
		"files": ["**/*"]
	}
]
```

**Comment by PeterDaveHello** [comment]
Use this:
```js
"npmFileMap": [
    {
        "basePath": "dist",
        "files": [
          "**/*"
        ]
    }
]
```

**Comment by Craga89** [comment]
As I'm doing this though the web interface, I'll simply re-apply the patch in another branch instead as cloning the whole repo is taking ages! Will re-open

**Pull request closed without merging**